puppet/.librarian/puppet/config

```diff
@@ -0,0 +1,3 @@
+---
+LIBRARIAN_PUPPET_DESTRUCTIVE: 'false'
```

Comment:
I don't know but it's the default :p

/me clones the repo to grep for the answer :sweat_smile: